hypothesis-python/src/hypothesis/ghostwriter.py

```diff
@@ -156,6 +228,12 @@ def _make_test(
         test_body=indent(test_body, prefix="    "),
     ).replace("builtins.", "")
 
+    if style == "unittest":
+        imports.add("unittest")
+        body = "class Test{}{}(unittest.TestCase):\n".format(
```

Comment:
You don't have to do this, but I wonder if you this would be better handled using the pretty printer module we have vendored? It's good for managing indentation and that sort of thing.